### PR TITLE
Kelnera/sidebar announcements

### PR DESCRIPTION
**Pull request by kelnera**
@osu-cass/cass-web This is the Sidebar Announcement feature in action. Test it out and let me know if it works for you. We had some issues so I want to make sure this is functional.

### CODE REVIEW

**Comment by kelnera** [comment]
Also, my apologies, but I don't remember how to stop my desktop from picking up all those commits. It just does it automatically for some reason. That last commit called "Add F&F Announcement" is what you want.

File: content/news/posts/AnnounceOpenHouse.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CASS Friends and Family Reception
+=================================
+:date: 2016-10-26
```
Comment (leian7) [inline content/news/posts/AnnounceOpenHouse.rst]: Fixed: Used to be 2016-26-10. The announcement now correctly shows up in the sidebar.

**Comment by leian7** [comment]
+1

**Comment by kelnera** [comment]
@leian7 @athai Okay, the date carried over to my branch, but now the announcement is not showing up for me when I build it on localhost. Let's just say this isn't going on the website in time, but we should still work on this.

**Comment by athai** [comment]
@kelnera Everything seems to be working fine for me though? Are you checking out the right branch in the dougfir-pelican-theme? 

``develop`` should be up-to-date. Try running, ``cd dougfir-pelican-theme``, then ``git branch`` to see what branch you're on. If you're already on ``develop``, run ``git pull origin develop`` in your terminal to update the branch. Otherwise, you'll need to run ``git checkout develop`` first.

**Comment by kelnera** [comment]
@athai Yes, I checked that and it still wasn't working. But it sounds like it's working for everyone else so I'll merge and hopefully it will show up on the staging site.

**Pull request merged**